src/types/enums.types.ts

```diff
@@ -70,7 +71,8 @@ export enum TransactionsDefaultTitles {
   success = 'Transaction successful',
   failed = 'Transaction failed',
   pending = 'Processing transaction',
-  timedOut = 'Transaction timed out'
+  timedOut = 'Transaction timed out',
+  invalid = 'Transaction invalid'
```

Comment:
can you add a comment on when a transaction is considered invalid? since it's not a server status

Reply:
actually is a transaction status